woocommerce/woocommerce
# [Experimental] Product Filters Redesign > Overlay Navigation: Add Block to the Product Filters Block

### Changes proposed in this Pull Request:





Closes #49690 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

> [!NOTE]  
> **Pre-requisite**
> - If you're using WC Beta Tester plugin, make sure the `experimental-blocks` feature is enabled (Tools > WCA Test Helper > Features).
> - Make sure to reset the Product Filters (Experimental) template part.

1. Navigate to Product Catalog template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filters (Experimental)" template part in the block inserter. Add it to your content.
5. Open the list view and make sure the Navigation block is added to the Product Filters (Experimental) block.
6. Select the Product Filters (Experimental) block and test the different Overlay settings: Never, Mobile, and Always.
    6.1. When Never is selected, the Navigation block should be hidden in the editor and in the Frontend.
    6.2. When Mobile is selected, the Navigation block should appear only when Mobile is selected.
    6.3. When Always is selected, the Navigation block should always appear. 



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment  Add the Product Filters Overlay Navigation block to the Product Filters block.

</details>
